From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rick Yiu <rickyiu@google.com>
Date: Fri, 16 Oct 2020 23:01:16 +0800
Subject: ANDROID: sched: Export symbols for vendor EAS wakup path function

It needs to export some symbols so that vendor module could reference.

Bug: 170507310
Signed-off-by: Rick Yiu <rickyiu@google.com>
Change-Id: I1b3c3ea8d0a11c01a1ca9e124e93f85e52856dc4
(cherry picked from commit a61271a41a2c2825a51bd7655fc446e16d23f5f6)
Signed-off-by: Will McVicker <willmcvicker@google.com>
---
 kernel/sched/pelt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/sched/pelt.c b/kernel/sched/pelt.c
--- a/kernel/sched/pelt.c
+++ b/kernel/sched/pelt.c
@@ -306,6 +306,7 @@ int __update_load_avg_blocked_se(u64 now, struct sched_entity *se)
 
 	return 0;
 }
+EXPORT_SYMBOL_GPL(__update_load_avg_blocked_se);
 
 int __update_load_avg_se(u64 now, struct cfs_rq *cfs_rq, struct sched_entity *se)
 {
